analyzer/tests/unit/test_checkers.py

```diff
@@ -1,79 +0,0 @@
-# -------------------------------------------------------------------------
```

Comment:
You removed the `available` function but you introduced a different one: `check_available_checkers`. So in this case it would be better to test this new function instead of dropping the whole test case.

Reply:
The new function can't be tested that easy, because this new version requires the analyzers' config handlers which is unreasonably difficult to mock. On the other hand every other analyzer test goes through this function because it is called in `perform_analysis()` function, so it is implicitly tested many times. But if you have some other suggestion on its testing, please, share it.